it/src/test/java/org/eclipse/jkube/integrationtests/wildfly/jar/WildflyJarOcITCase.java

```diff
@@ -0,0 +1,128 @@
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.integrationtests.wildfly.jar;
+
+import io.fabric8.kubernetes.client.DefaultKubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.openshift.api.model.ImageStream;
+import io.fabric8.openshift.client.OpenShiftClient;
+import org.apache.maven.shared.invoker.InvocationResult;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.parallel.ResourceLock;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.List;
+
+import static org.eclipse.jkube.integrationtests.Locks.CLUSTER_RESOURCE_INTENSIVE;
+import static org.eclipse.jkube.integrationtests.OpenShift.cleanUpCluster;
+import static org.eclipse.jkube.integrationtests.Tags.OPEN_SHIFT;
+import static org.eclipse.jkube.integrationtests.assertions.YamlAssertion.yaml;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.anEmptyMap;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.jupiter.api.parallel.ResourceAccessMode.READ_WRITE;
+
+@Tag(OPEN_SHIFT)
+@TestMethodOrder(OrderAnnotation.class)
+class WildflyJarOcITCase extends WildflyJar {
+
+  private OpenShiftClient oc;
+
+  @BeforeEach
+  void setUp() {
+    oc = new DefaultKubernetesClient().adapt(OpenShiftClient.class);
+  }
+
+  @AfterEach
+  void tearDown() {
+    oc.close();
+    oc = null;
+  }
+
+  @Override
+  public KubernetesClient getKubernetesClient() {
+    return oc;
+  }
+
+  @Override
+  protected List<String> getProfiles() {
+    return Collections.singletonList(OPEN_SHIFT);
+  }
+
+  @Test
+  @Order(1)
+  @ResourceLock(value = CLUSTER_RESOURCE_INTENSIVE, mode = READ_WRITE)
+  @DisplayName("oc:build, should create image")
+  void ocBuild() throws Exception {
+    // When
+    final InvocationResult invocationResult = maven("oc:build");
+    // Then
+    assertThat(invocationResult.getExitCode(), Matchers.equalTo(0));
+    final ImageStream is = oc.imageStreams().withName("wildfly-jar-microprofile").get();
+    assertThat(is, notNullValue());
+    assertThat(is.getStatus().getTags().iterator().next().getTag(), equalTo("latest"));
+  }
+
+  @Test
+  @Order(1)
+  @DisplayName("oc:resource, should create manifests")
+  void ocResource() throws Exception {
+    // When
+    final InvocationResult invocationResult = maven("clean oc:resource"); //Clean to free up DiskSpace
```

Comment:
Clean is not needed here (I guess you got it from Thorntail's test, but must be removed from there too> no longer needed)
```suggestion
    final InvocationResult invocationResult = maven("oc:resource"); //Clean to free up DiskSpace
```

Reply:
Thank-you, removed.